doc/source/merging.rst

```diff
@@ -817,6 +817,12 @@ equivalent:
     left.join(right, on=key_or_keys)
     pd.merge(left, right, left_on=key_or_keys, right_index=True,
           how='left', sort=False)
+          
+.. warning::
+
+   Joining on keys with duplicate values when joining large dataframes would cause severe memory overflow, sometimes freezes the
```

Comment:
@xgdgsc well, reading what you just did, it is completely non-obvious what you mean. an example is worth 1000 words. And when I mean example, I mean about 2 lines of code.